package.json

```diff
@@ -11,7 +11,8 @@
     "webapp"
   ],
   "dependencies": {
-    "immutable": "4"
+    "immutable": "4",
+    "ts-mockito": "^2.6.1"
```

Comment:
the root package.json is hardly ever changed, especially for dev deps used in a single package. I use it for syncing version of dependencies that are needed across many packages.
please modify discojs/package.json if you want to add a package for discojs. (which you probably don't need as stated previously)